vendor/github.com/siddontang/goredis/conn.go

```diff
@@ -33,7 +33,7 @@ func Connect(addr string) (*Conn, error) {
 }
 
 func ConnectWithSize(addr string, readSize int, writeSize int) (*Conn, error) {
```

Comment:
I prefer defining a Options contains addr, read/writeSize and connectTimeout, etc..., then add a function ConnectWithOptions